### PR TITLE
CodeQL recommendations

### PR DESCRIPTION
**Pull request by dacbd**

### CODE REVIEW

File: src/drivers/github.js [hunk 1]
```diff
@@ -66,11 +66,11 @@ const octokit = (token, repo) => {
...
     }
   };
-
-  if (!repo.includes('github.com')) {
+  const { host, hostname } = new url.URL(repo);
+  if (hostname !== 'github.com') {
     // GitHub Enterprise, use the: repo URL host + '/api/v3' - as baseURL
     // as per: https://developer.github.com/enterprise/v3/enterprise-admin/#endpoint-urls
-    const { host } = new url.URL(repo);
+
```
Comment (dacbd) [inline src/drivers/github.js:73]:
```suggestion
```

**Review by 0x2b3bfa0 — DISMISSED** [review]
💯

File: bin/cml/asset/publish.js [hunk 2]
```diff
@@ -15,8 +15,10 @@ exports.handler = async (opts) => {
     opts.native = true;
   }
 
-  const { file, repo, native, asset: path } = opts;
-  const cml = new CML({ ...opts, repo: native ? repo : 'cml' });
+  // const { file, repo, native, asset: path } = opts;
+  const { file, asset: path } = opts;
+  // const cml = new CML({ ...opts, repo: native ? repo : 'cml' });
+  const cml = new CML({ ...opts });
```
Comment (dacbd) [inline bin/cml/asset/publish.js:21]:
```suggestion
  const { file, asset: path } = opts;
  const cml = new CML({ ...opts });
```

**Comment by dacbd** [comment]
well CodeQL caught a slightly related bug :upside_down_face:

**Review by 0x2b3bfa0 — APPROVED** [review]

**Pull request merged**